arangod/Aql/ExecutionBlockImpl.cpp

```diff
@@ -272,8 +272,15 @@ std::unique_ptr<OutputAqlItemRow> ExecutionBlockImpl<Executor>::createOutputRow(
 }
 
 template<class Executor>
-Executor& ExecutionBlockImpl<Executor>::executor() {
-  return _executor;
+auto ExecutionBlockImpl<Executor>::executor() noexcept -> Executor& {
+  TRI_ASSERT(_executor.has_value());
+  return *_executor;
+}
```

Comment:
```suggestion
auto& ExecutionBlockImpl<Executor>::executor() noexcept {
  TRI_ASSERT(_executor.has_value());
  return *_executor;
}
```

Reply:
I'd prefer that, too, but it doesn't match the declaration.